.github/workflows/release_flow.yml

```diff
@@ -169,8 +169,8 @@ jobs:
       uses: svenstaro/upload-release-action@v2
       with:
         repo_token: ${{ steps.token.outputs.token }}
-        file: ./codecov-cli/dist/codecovcli_${{ matrix.distro_name }}_${{ matrix.arch }}
-        asset_name: codecovcli_${{ matrix.distro_name }}_${{ matrix.arch }}
+        file_glob: true
+        file: ./codecov-cli/dist/codecovcli_*
```

Comment:
if using file_glob, asset_name doesn't apply. Will just use filename, which is fine